src/NLog.Targets.Seq/SeqTarget.cs

```diff
@@ -46,6 +46,13 @@ public sealed class SeqTarget : Target
         /// </summary>
         public int JsonPayloadMaxLength { get; set; }
 
+        /// <summary>
+        /// The minimum log level that should be sent to the server.
+        /// This may be adjusted dynamically in response to server hints.
+        /// </summary>
+        public LogLevel MinimumLevel { get; set; } = LogLevel.Trace;
+
+
```

Comment:
nit: extra space here

Reply:
"Space: the character often mistaken for the absence of a character."

I spent ages looking at this and thinking "what space is he talking about". And then I realized you mean the blank line ;-) I will remove it.